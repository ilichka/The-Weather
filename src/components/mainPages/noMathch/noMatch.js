import "./index.scss";

export default () => {
  const noMatch = document.createElement("div");
  noMatch.classList.add("no-match-page");
  noMatch.innerHTML = `404 Page not found`;
  return noMatch;
};
