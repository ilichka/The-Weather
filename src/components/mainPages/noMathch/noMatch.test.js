import noMatch from "./noMatch";

describe("No match page snapshot", () => {
  it("Renders correctly template", () => {
    const template = noMatch();

    expect(template).toMatchSnapshot();
  });
});
