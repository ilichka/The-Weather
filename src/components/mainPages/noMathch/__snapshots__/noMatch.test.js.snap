// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No match page snapshot Renders correctly template 1`] = `
<div
  class="no-match-page"
>
  404 Page not found
</div>
`;
